app/routes/events/components/EventAdministrate/RegistrationTables.tsx

```diff
@@ -169,14 +200,29 @@ export class RegisteredTable extends Component<Props> {
       handlePresence,
       handlePayment,
       handleUnregister,
-      clickedUnregister,
+      showPresence,
       showUnregister,
       event,
       pools,
     } = this.props;
+
+    const gradeColumn = {
+      title: 'Klassetrinn',
+      dataIndex: 'user.grade',
+      render: GradeRenderer,
+      sorter: (a, b) => {
+        if (a.user.grade && b.user.grade) {
+          if (a.user.grade.name > b.user.grade.name) return 1;
```

Comment:
I don't know that much about js sorting internals, but it might be beneficial to return 0 on equal grades so it always keeps its original. It already does in my browser, so this is really nitpicky but got to use algdat for something right :shrug:

Reply:
I didn't want to touch any of this logic, but sure I can add that.